typo3/cms-core/CVE-2022-36108.yaml

```diff
@@ -0,0 +1,11 @@
+title: 'TYPO3-CORE-SA-2022-010: Cross-Site Scripting in &lt;f:asset.css&gt; view helper'
```

Comment:
I'm not sure about the encoding here? Should it be plain instead?

Reply:
yes, this should be plain text, not HTML entities.